### ISSUE TITLE
Failed to deploy distro docker-desktop

### ISSUE THREAD

**Issue by blyind** [docker/for-win#12082]
- [x ] I have tried with the latest version of Docker Desktop
  - [x ] I have tried disabling enabled experimental features
  - [x ] I have uploaded Diagnostics
  - Diagnostics ID:7EFD7137-4077-4AA3-A708-0D4EDD4F0EB2/20210904153652

### Actual behavior
failed to deploy distro docker-desktop

### Expected behavior
docker-desktop would start correctly after a fresh install

### Information

  - Is it reproducible?  
          yes every time i uninstall, and fresh install its the same issue
  - Is the problem new?  
         its never worked for me
  - Did the problem appear with an update? 
         only tried with current version
  - A reproducible case if this is a bug, Dockerfiles with reproduction inside is best.
  

  - Windows Version:  windows 11 version 21H2 OS Build 22000.176
  - Docker Desktop Version:
  - WSL2 or Hyper-V backend?  
        WSL2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM:
        no

**Comment by simpleauthority** [comment]
This is happening for me as well. I can't get docker desktop to run, and thereby neither will my WSL2 containers.

**Comment by simpleauthority** [comment]
Installing the latest Windows update and rebooting fixed the issue. Docker Desktop is now working as expected. Not sure if you have tried this yet, or not.

**Comment by blyind** [comment]
> Installing the latest Windows update and rebooting fixed the issue. Docker Desktop is now working as expected. Not sure if you have tried this yet, or not.

I've updated to the latest build, but I'm running a windows 11 insiders beta channel build.
I'm guessing that is why I'm having such a hard time getting docker-desktop to run.
I think my only option is to do a total format and clean install of windows 10 as from what I have read thats the only way to go back.

   I've spent like 3 days trying literally everything I can think of or could find on google  and thats probably it. About time I admit defeat on this one.
